modules/core/server/controllers/core.routes.controller.js

```diff
@@ -457,4 +449,49 @@ exports.setCRUDRoutes = function (app, basename, DBClass, policy, which, policym
 		}));
 };
 
+function runPaginate(model, options) {
+	var query = {};
+	var filter = {};
+	var skip = 0;
+	var limit = 100;
+	var sortby = {};
+
+	// fields, populate, and userCan aren't specific to table params
+	var fields = null;
+	var populate = null;
+	var userCan = false;
 
+	if (options) {
+		if (options.filterBy) {
+			query = options.filterBy;
+		}
+		if (options.filterByFields) {
+			for (var key in options.filterByFields) {
+				if (options.filterByFields.hasOwnProperty(key)) {
+					filter[key] = options.filterByFields[key];
+					console.log(filter);
+				}
+			}
+		}
+		try {
+			limit = parseInt(options.limit);
+			skip = parseInt(options.start);
+		} catch(e) {
+
```

Comment:
This code is inherited. I don't know why the original developer did this. If anything let's put a console.log output here. ?